packages/core/src/components/button/_button.scss

```diff
@@ -49,6 +49,7 @@ Styleguide components.button.css
 .pt-button {
   @include pt-button-base();
   @include pt-button-height($pt-button-height);
+  position: relative;
```

Comment:
Is that fine? I wonder if it could be breaking in some cases. Would need confirmation from Gilad/Chris/Adi